mapping/sustainability.yaml

```
@@ -1576,14 +1564,14 @@
     mapping: |-
         Contracting process level:
 
-        Publish a summary of the labor obligations:
+        1. Publish a summary of the labor obligations:
 
-        1. For each labor obligation in the contract, add a code from the [laborObligations](../../reference/codelists.md#laborobligations) codelist to the`.summary.social.laborObligations.obligations` array.
-        2. Optionally, add a further explanation of the labor obligations to `.summary.social.laborObligations.description`.
+           1. For each labor obligation, add a code from the open [laborObligations](../../reference/codelists.md#laborobligations) codelist to the [`.summary.social.laborObligations.obligations`](project-schema.json,/definitions/ContractingProcessSummary,social/laborObligations/obligations) array.
```

```suggestion
           1. For each labor obligation, add the relevant code from the open [laborObligations](../../reference/codelists.md#laborobligations) codelist to the [`summary.social.laborObligations.obligations`](project-schema.json,/definitions/ContractingProcessSummary,social/laborObligations/obligations) array.
```

"add *a* code..." feels a bit ambiguous, I don't think this was picked up in the PR that created this initially.

---

Done in 73bc7abfdb6a4bf562298f1f80ac2d6cfd7656ec